api/src/pdc/services/territory/config/meilisearch.ts

```diff
@@ -0,0 +1,6 @@
+export const config = {
+  host: process.env.APP_MEILISEARCH_HOST || 'http://localhost',
+  apiKey: process.env.APP_MEILISEARCH_APIKEY || '',
+  index: process.env.APP_MEILISEARCH_INDEX || 'geo',
+  batchSize: Number(process.env.APP_MEILISEARCH_BATCH) || 1000,
+}
```

Comment:
```suggestion
import { env } from '@ilos/core';

export const host = env.or_fail('APP_MEILISEARCH_HOST', 'http://localhost');
export const apiKey = env.or_fail('APP_MEILISEARCH_APIKEY');
export const index = env.or_fail('APP_MEILISEARCH_INDEX', 'geo');
export const batchSize = env.or_int('APP_MEILISEARCH_BATCH', 1000);
```

Reply:
Tu as des helpers pour récupérer les variables d'environnement et les typer comme il faut.
En plus, je te conseille d'exporter directement les variables sans passer par un objet `config` pour que ce soit plus court :
`const val = this.config.get('meilisearch.host')`
plutôt que `const val = this.config.get('meilisearch.config.host')`

Reply:
Attention, je n'ai pas fait de modifs sur l'utilisation des clés de config dans le code